spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -412,16 +412,17 @@ version: '3.5'
 
 services:
   localstack:
-    image: localstack/localstack:2.0.0
+    image: localstack/localstack:2.1.0
     environment:
+      - EAGER_SERVICE_LOADING=1
```

Comment:
While I'm OK with new script location, I don't understand a requirement for this property to be set.
Why do you think this project should enforce such a property?
I have just tested the project against this latest Locastack and everything works well.
In fact, all my tests really rely on a lazy behavior of the Localstack.
Previously I had to use different container instances for different tests scopes with their specific services.
Now I just need a single container instance for the whole test suite - and AWS services are started for me on demand.
So, why would one enforce this `EAGER_SERVICE_LOADING`, especially from this Kinesis binder project?
Why just don't leave this up to end-user to decide how to configure their test environment?

Reply:
Agreed, updated script.